src/node/createSetupServer.ts

```diff
@@ -149,7 +149,7 @@ export function createSetupServer(...interceptors: Interceptor[]) {
       /**
        * Stops requests interception by restoring all augmented modules.
        */
-      close() {
+      close(): void {
```

Comment:
I believe that both argument and return types are inferred from the `ServerInstance` return type you assign to the `setupServer` function declaration. There's no need to explicitly annotate their return types as `void`.